tests/default/indices/upgrade.yaml

```diff
@@ -0,0 +1,40 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test upgrading index using both GET and POST methods.
+epilogues:
+  - path: /movies
+    method: DELETE
+    status: [200, 404]
+prologues:
+  - path: /_bulk
+    method: POST
+    parameters:
+      refresh: true
+    request:
+      content_type: application/x-ndjson
+      payload:
+        - {create: {_index: movies, _id: movie1}}
+        - {director: Bennett Miller, title: The Cruise, year: 1998}
+        - {create: {_index: movies, _id: movie2}}
+        - {director: Nicolas Winding Refn, title: Drive, year: 1960}
+chapters:
+  - synopsis: Check index upgrade status (GET).
```

Comment:
Probably good to flip this one with the triggering of an upgrade so we can get actual results and check schema.

Reply:
If I understood you correctly, then I have corrected